tutorial/list-with-data.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: page
+title: List of struct and ListWithData widget
+---
+
+
+Sometime, you need to display a list of complex item, more complex than a list of string, but you also want to rely on Fyne's databinding. It can be the case when you want to display a list of contact, or books for example.
+
+
+## Defining the binding
+
+
+Well first thing first, let's start by defining our binding in our app : 
+
+```go
+var ContactListBinding binding.UntypedList = binding.NewUntypedList()
```

Comment:
best not to export variables unless required

Reply:
Also here you should define the struct `contact` or whatever.
For the rest of the example can you use a local type rather than `contact.Contact`?